code/modules/mob/living/carbon/human/species/slime.dm

```diff
@@ -85,6 +85,11 @@
 
 /datum/species/slime/handle_life(mob/living/carbon/human/H)
 	// Slowly shifting to the color of the reagents
+	if((H.blood_color != BlendRGB(H.skin_colour, "#acacac", 0.5))||H.blood_color == null) // Put here, so if it's a roundstart, dyed, or CMA'd slime, their blood changes to match skin
```

Comment:
```suggestion
	if(H.blood_color == null || H.blood_color != BlendRGB(H.skin_colour, "#acacac", 0.5)) // Put here, so if it's a roundstart, dyed, or CMA'd slime, their blood changes to match skin
```
spacing, a () to many and order of things. Cheaper to check null than doing the blend check